src/components/HornedBeast.js

```diff
@@ -1,17 +1,29 @@
 import {Component} from 'react';
-import heart from './images/heart.png';
+import heart from '../images/heart.png';
+import SelectedBeast from './SelectedBeast.js';
 
 class HornedBeast extends Component {
 
   constructor(props) {
     super(props);
     this.state = {
       timesClicked: 0,
+      showModal: false
     };
   }
 
   handleClick = () => {
     this.setState({timesClicked: this.state.timesClicked + 1});
+    this.setState({showModal: true});
```

Comment:
Hi there!

To answer your question on canvas, you can indeed add multiple things to one setState call (since it is an object).
Ex:
```
this.setState({timesClicked: this.state.timesClicked + 1}, {showModal: true});
```

Reply:
Updated comment here for setState (passing more than one thing in. I forgot to remove extra curlies previously oops :)